agnostic-svelte-ts/src/lib/components/Tabs/TabButtonCustom.svelte

```diff
@@ -0,0 +1,77 @@
+<style>
+	.buttonWrap {
+		opacity: 1;
+		transition: opacity 0.2s;
+		margin: 0;
+	}
+	.buttonWrap:not(:first-of-type) {
+		/* So we don't get a "double border" between tab buttons */
+		margin-inline-start: -1px;
+	}
+	.buttonWrap:hover {
+		opacity: 0.7;
+	}
+	.buttonWrap:hover:focus {
+		opacity: 1;
+	}
+	.tabButton:focus {
+		box-shadow: 0 0 0 var(--agnostic-focus-ring-outline-width)
+			var(--agnostic-focus-ring-color);
+		outline: var(--agnostic-focus-ring-outline-width)
+			var(--agnostic-focus-ring-outline-style)
+			var(--agnostic-focus-ring-outline-color);
+		transition: box-shadow var(--agnostic-timing-fast) ease-out;
+	}
+	.active .tabButton {
+		outline: 1px solid var(--agnostic-primary-hover);
+	}
+</style>
+
+<script>
```

Comment:
Shouldn't this be `<script lang="ts">`?

Reply:
Whoops, I forgot to update this, will push it soon! 👍